### PR TITLE
Export import keys using CLI

### PR DESCRIPTION
**Pull request by tomasmik**
You can now:
* Export to file/terminal
* Import from file/input

Imported identities are set as default.

### CODE REVIEW

File: cmd/commands/cli/command_identities.go [hunk 1]
```diff
@@ -310,3 +320,85 @@ func (c *cliApp) setBeneficiary(actionArgs []string) {
...
+			return err
+		}
+
+		err := write()
+		if err == nil {
+			clio.Success("Identity exported to file:", filepath)
+			return
+		}
+
+		clio.Warn("Export succeeded, file write failed. Will print to screen.")
```
Comment (soffokl) [inline cmd/commands/cli/command_identities.go:370]: This `Will print to screen` could be a problem. If the user does not expect private keys to be printed to stdout we should not do it. If something failed the user can do a second attempt or print to stdout manually.

File: cmd/commands/cli/command_identities.go [hunk 2]
```diff
@@ -310,3 +320,85 @@ func (c *cliApp) setBeneficiary(actionArgs []string) {
...
+	}
+
+	clio.Success("Private key exported: ")
+	fmt.Println(string(blob))
+}
+
+const usageImportIdentity = "import <passphrase> <key-string/key-file>"
```
Comment (soffokl) [inline cmd/commands/cli/command_identities.go:377]:
```suggestion
const usageImportIdentity = "import <passphrase> <path/to/key-file>"
```

File: cmd/commands/cli/command_identities.go [hunk 3]
```diff
@@ -310,3 +320,85 @@ func (c *cliApp) setBeneficiary(actionArgs []string) {
...
+	}
+
+	key := actionsArgs[1]
+	passphrase := actionsArgs[0]
+
+	blob := []byte(key)
+	if _, err := os.Stat(key); err == nil {
+		blob, err = ioutil.ReadFile(key)
+		if err != nil {
+			clio.Error(fmt.Sprintf("cant read provided file: %s reason: %s", key, err.Error()))
```
Comment (soffokl) [inline cmd/commands/cli/command_identities.go:392]:
```suggestion
			clio.Error(fmt.Sprintf("can't read provided file: %s reason: %s", key, err.Error()))
```

**Review by soffokl — APPROVED** [review]

File: cmd/commands/cli/command_identities.go (hunk 2 above)
Comment (tomasmik) [reply cmd/commands/cli/command_identities.go:377]: You can give a string as in paste your key, or a file, so I think im correct here.

**Review by vkuznecovas — APPROVED** [review]

**Pull request merged**